pages/apple-silicon/how-to/update-os-mac-mini.mdx

```diff
@@ -72,6 +75,10 @@ To manually update the operating system using the `softwareupdate` tool, follow
         ```
     </Message>
 
+    <Message type="important">
+      Your Mac mini may restart during the upgrade process and be unavailable while the software update is installed. Once the update is successfully completed, it will become available again.### Upgrading the OS using SSH
```

Comment:
```suggestion
      Your Mac mini may restart during the upgrade process and be unavailable while the software update is installed. Once the update is successfully completed, it will become available again.
```